tests/tacacs/utils.py

```diff
@@ -65,27 +65,24 @@ def log_exist(ptfhost, sed_command):
     pytest_assert(exist, "Not found data: {} in tacplus server log".format(data))
 
 
-def get_auditd_config_reload_timestamp(duthost):
+def get_auditd_config_reload_line_count(duthost):
     res = duthost.shell("sudo journalctl -u auditd --boot --no-pager | grep 'audisp-tacplus re-initializing configuration'") # noqa E501
     logger.info("aaa config file timestamp {}".format(res["stdout_lines"]))
 
-    if len(res["stdout_lines"]) == 0:
-        return ""
+    return len(res["stdout_lines"])
 
-    return res["stdout_lines"][-1]
 
-
-def change_and_wait_aaa_config_update(duthost, command, last_timestamp=None, timeout=10):
-    if not last_timestamp:
-        last_timestamp = get_auditd_config_reload_timestamp(duthost)
+def change_and_wait_aaa_config_update(duthost, command, last_line_count=None, timeout=10):
```

Comment:
i just have 1 Q: did we confirm this function is only used in `tests/tacacs/test_accounting.py`? because the function name is changed, we need to make sure other callers also change

Reply:
just did a quick search and it shows besides test_accounting,py, there is test_ro_disk.py and test_authorization.py that call this function, please change accordingly.